src/pgw_s5s8.erl

```diff
@@ -378,8 +383,9 @@ handle_request(ReqKey,
     %% TBD: reselect VRF and Pool based on outcome of authenticate
     %% -----------------------------------------------------------
 
-    ActiveSessionOpts1 = apply_session_opts(APNOpts, ActiveSessionOpts0),
-    {IPOpts, ContextPending} = assign_ips(ActiveSessionOpts1, PAA, ContextVRF),
+    ActiveSessionOpts1 = apply_apn_defaults(APNOpts, ActiveSessionOpts0),
+    ContextAddTimeout = add_apn_timeout(APNOpts, ContextVRF),
```

Comment:
should use the SessionOpts instead of the APNOpts, that way AAA can overwrite the Idle-Timeout